coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -186,13 +238,38 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
         metadata = cls.Metadata(*(str(docstyle_settings.get(req_setting, ''))
                                   for req_setting in metadata_settings))
 
+        try:
+            class_padding = cls.ClassPadding(
+                *(int(padding) for padding in tuple(
```

Comment:
Hmm now even I m wondering the same. You are correct about `tuple('1,0')` will be `("1", ",", " ", "0")` but what I figured out is `docstyle_settings` is a `Setting` object. Now when I do `tuple(docstyle_settings['class_padding'].value)` gives me `("1", ",", " ", "0")` but when I only do `tuple(docstyle_settings['class_padding'])` gives me `('1', '0')`(some magic :stuck_out_tongue: ). So I think let it be that way.